qa/rpc-tests/p2p-versionbits-warning.py

```diff
@@ -73,7 +76,10 @@ def setup_network(self):
         self.nodes.append(start_node(0, self.options.tmpdir, self.node_options))
 
         import re
-        self.vb_pattern = re.compile("^Warning.*versionbit")
+        # bip135 begin
+        # with new unknown version algo, warning has changed.
+        self.vb_pattern = re.compile("^Warning: Unknown block versions being mined Its possible unknown rules are in effect")
```

Comment:
EDIT: Turned out the actual warning is run through SanitizeString() as part of the command, that's why the pattern in the test was different.